code/obj/item/misc_weapons.dm

```diff
@@ -1717,6 +1717,122 @@ obj/item/whetstone
 		throwforce = 20 //higher base damage, lower once the slasher starts scaling up their machete
 		force = 20
 
+
+// Halberd- Experimental weapon by NightmareChamillian
+/obj/item/halberd
+	name = "Halberd"
+	desc = "An ancient axe-like weapon capable of cleaving and piercing flesh with ease. You have no idea what this is doing outside a museum."
+	icon = 'icons/obj/large/64x32.dmi'
+	icon_state = "halberd"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	item_state = "halberd1"
+
+	w_class = W_CLASS_BULKY
+	two_handed = 1
+	throw_range = 10
+	throwforce = 30 //yeet like spear
+	stamina_crit_chance = 5
+
+	//these combat variables change depending on intent- starts with help intent vars
+	force = 28
```

Comment:
A bunch of these should be defines, pretty much anything which is repeated. Don't need to define every thing on every mode but if you're referencing a number or string on the same mode more than once make it into a define.
Just defining the defaults is Probably Fine
(case in point, this is 28 even though the comment indicates it should be 24)

(syntax: just put `#define HALBERD_STAT_THING 420` above the type definition and `#undef HALBERD_STAT_THING_420` below all the procs.)